Pester.psm1

```diff
@@ -829,45 +835,59 @@ New-PesterOption
                 param (
                     [Parameter(Position = 0)]
                     [string] $Path,
-
+                    [string] $Script, #in order to allow passing test script as a string
```

Comment:
An added parameter is not explained in the comment based help.

Reply:
I removed it from here. Mentioned usage of this parameter in example (line 661). Do you think it understandable enough?

Reply:
@it-praktyk Could you please review my PR again? I`ve added test and description for parameter. Changes are backward compatible.